### PR TITLE
fix: move tag '</a>' in view post to locale.yml

### PR DESCRIPTION
**Pull request by TiagoDanin**
image
This can cause problems for translators in Crowdin.

### CODE REVIEW

**Comment by BinaryMuse** [comment]
Should the period get moved up as well? That seems to be the pattern. I'll submit a couple suggested changes.

File: data/locale.yml [hunk 1]
```diff
@@ -220,7 +220,7 @@ blog:
   description: All the latest news from the Electron team and community.
   # Post show page specific localization strings
   feedback: Have feedback on this post? Let <a href="https://twitter.com/electronjs">@electronjs</a> know on Twitter.
-  help: Need help or found a bug? <a href="/contact">Contact us
+  help: Need help or found a bug? <a href="/contact">Contact us</a>
```
Comment (BinaryMuse) [inline data/locale.yml]:
```suggestion
  help: Need help or found a bug? <a href="/contact">Contact us</a>.
```

File: views/layouts/post.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <div class="bg-shade p-4 rounded-1 mt-6 mt-lg-8">
       <p>{{{localized.blog.feedback}}}</p>
-      <p>{{{localized.blog.help}}}</a>.</p>
+      <p>{{{localized.blog.help}}}.</p>
```
Comment (BinaryMuse) [inline views/layouts/post.html]:
```suggestion
      <p>{{{localized.blog.help}}}</p>
```

Comment (TiagoDanin) [reply views/layouts/post.html]: :+1:

**Pull request merged**